vmm/src/cpu.rs

```diff
@@ -384,6 +455,65 @@ impl Vcpu {
             ts.as_micros()
         );
     }
+
+    #[allow(unused)]
```

Comment:
I don't see these being removed in a later commit.

Reply:
Yes it's removed here: https://github.com/cloud-hypervisor/cloud-hypervisor/pull/998/commits/a7c6560347f302fcab7826900c7e086f570f4b1f#diff-39805f31fede77dcaa77b9055d65e821L459